src/Test/Utilities/Portable/Mocks/TestReferences.cs

```diff
@@ -10,2663 +11,1114 @@ public static class MetadataTests
-            {
-                private static PortableExecutableReference s_v1MTTestLib1;
-                public static PortableExecutableReference netmodule
-                {
-                    get
-                    {
-                        if (s_v1MTTestLib1 == null)
-                        {
-                            s_v1MTTestLib1 = ModuleMetadata.CreateFromImage(TestResources.SymbolsTests.V3.MTTestModule4).GetReference(display: "MTTestModule4.netmodule");
-                        }
-
-                        return s_v1MTTestLib1;
-                    }
-                }
-            }
-        }
-
-        public static class MultiModule
-        {
-            private static PortableExecutableReference s_assembly;
-            public static PortableExecutableReference Assembly
-            {
-                get
-                {
-                    if (s_assembly == null)
-                    {
-                        s_assembly = AssemblyMetadata.Create(
-                            ModuleMetadata.CreateFromImage(TestResources.SymbolsTests.MultiModule.MultiModuleDll),
-                            ModuleMetadata.CreateFromImage(TestResources.SymbolsTests.MultiModule.mod2),
-                            ModuleMetadata.CreateFromImage(TestResources.SymbolsTests.MultiModule.mod3)).GetReference(display: "MultiModule.dll");
-                    }
-
-                    return s_assembly;
-                }
-            }
-
-            private static PortableExecutableReference s_mod2;
-            public static PortableExecutableReference mod2
-            {
-                get
-                {
-                    if (s_mod2 == null)
-                    {
-                        s_mod2 = AssemblyMetadata.CreateFromImage(TestResources.SymbolsTests.MultiModule.mod2).GetReference(display: "mod2.netmodule");
-                    }
-
-                    return s_mod2;
-                }
-            }
-
-            private static PortableExecutableReference s_mod3;
-            public static PortableExecutableReference mod3
-            {
-                get
-                {
-                    if (s_mod3 == null)
-                    {
-                        s_mod3 = AssemblyMetadata.CreateFromImage(TestResources.SymbolsTests.MultiModule.mod3).GetReference(display: "mod3.netmodule");
-                    }
-
-                    return s_mod3;
-                }
-            }
-
-            private static PortableExecutableReference s_consumer;
-            public static PortableExecutableReference Consumer
-            {
-                get
-                {
-                    if (s_consumer == null)
-                    {
-                        s_consumer = AssemblyMetadata.CreateFromImage(TestResources.SymbolsTests.MultiModule.Consumer).GetReference(display: "Consumer.dll");
-                    }
-
-                    return s_consumer;
-                }
-            }
-        }
-
```

Comment:
Strange indentation is strange.

Reply:
(applies to this entire file)

Reply:
Sounds like you're suggesting the IDE provide some way to normalize the formatting :)